src/test/resources/examples/operationsSchema/controllers/spring/Controllers.kt

```diff
@@ -0,0 +1,122 @@
+package examples.operationsSchema.controllers
+
+import examples.operationsSchema.models.ArrayPostRequestBody
```

Comment:
**question (blocking):** I don't see this in the models file

Reply:
That's not a question 😛 

Should be fixed by https://github.com/cjbooms/fabrikt/pull/201 and was left out of this PR intentionally.

Reply:
I see. How did you run the tests without it? 💭

Reply:
The tests don't necessarily need the generated files to compile, as they're not actually part of the test sources.